test/per_file_coverage.sh

```diff
@@ -9,7 +9,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/crypto:95.5"
 "source/common/event:95.6" # Emulated edge events guards don't report LCOV
 "source/common/filesystem/posix:96.3" # FileReadToEndNotReadable fails in some env; createPath can't test all failure branches.
-"source/common/http/http2:96.0"
```

Comment:
Coverage decrease is due to removing lines that were runtime guarded and tests were parameterized to cover them.